### ISSUE TITLE
player/device reporting issue

### ISSUE THREAD

**Issue by aniel300** [croneter/PlexKodiConnect#1650]
am having this issue similar to the one described here https://github.com/croneter/PlexKodiConnect/issues/764 where i have cloned a kodi instance and if i tried to play the same thing or different things plex dash will show it as one session/device etc. i have tried rebooting the server, changing the device name under pkc settings, closed and opened kodi, generating new unique id in pkc settings but issue still persist, thank you in advance.

**Comment by aniel300** [comment]
just checked with Tautulli, same issue still happening

**Comment by croneter** [comment]
Also make sure - on the Tautulli side, so to speak - that your Kodi devices use different hostnames!

**Comment by aniel300** [comment]
issue still happening, if i remember correctly ,y this didn't used to be like this. am using the same user for this test

**Comment by croneter** [comment]
Pfffff, there really shouldn't be any issue there! 😢 

Kindly share the DEBUG logs from both Kodi devices, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by aniel300** [comment]
do u have a discord, so i can send u the files, last time i shared log my plex was hacked

**Comment by croneter** [comment]
Ouch 🤕
Nope, but you could email them: croneter at gmail

**Comment by aniel300** [comment]
done

**Comment by croneter** [comment]
Yes. Got it thanks. I edited your message, deleted my email to prevent spam 😜

**Comment by aniel300** [comment]
ok, i understand

**Comment by croneter** [comment]
For whatever reason, your unique Plex device ID did not change:
Log of device 1:
```
NOTICE: PLEX.clientinfo: Unique device Id plex_client_Id loaded: 02e0c255-a1a0-4d88-9868-ccdf253c948a
```
Log of device 2:
```
NOTICE: PLEX.clientinfo: Unique device Id plex_client_Id loaded: 02e0c255-a1a0-4d88-9868-ccdf253c948a
```
These strings need to be different from each other. 

Workaround:
* On one of the 2 devices, navigate to the [Kodi userdata directory](https://kodi.wiki/view/Userdata)
* Edit the file `settings.xml` in the folder `addon_data/plugin.video.plexkodiconnect`
* Change the string in this line:
```
<setting id="plex_client_Id">02e0c255-a1a0-4d88-9868-ccdf253c948a</setting>
```
to something like this (so just a changed last letter `a` --> `b`
```
<setting id="plex_client_Id">02e0c255-a1a0-4d88-9868-ccdf253c948b</setting>
```

**Comment by croneter** [comment]
There's indeed a bug there, will fix it if I find the time

**Comment by aniel300** [comment]
am trying kodi 19 and the new pkc for it as a ts step, right now

**Comment by croneter** [comment]
Should finally be solved with PKC beta version 3.5.9

**Comment by aniel300** [comment]
thank u very much @croneter, i really appreciate it, keep up the good work

**Comment by aniel300** [comment]
was this implemented into Kodi Leia

**Comment by croneter** [comment]
Not yet

**Comment by aniel300** [comment]
ok

**Comment by emilflorisse** [comment]
Thanks, I had the same problem (just discovered this after cloning 10 devices :'( )
Changing the <setting id="plex_client_Id"> worked, only needed to restart Kodi (Android Shield TV 8.2.3, Android 9, Kodi 19.3) and voila.

Thanks for the advice and keep up the good work.
And don't let the bugs annoy you too much; It's amazingly good what you've build right here!

**Comment by aniel300** [comment]
where do u change "id="plex_client_Id". btw i found a equal or better alternative, i just haven't play with and kodi in a long time. nor the dev of this project fixed the issue for kodi 17

**Comment by aniel300** [comment]
> For whatever reason, your unique Plex device ID did not change: Log of device 1:
> 
> ```
> NOTICE: PLEX.clientinfo: Unique device Id plex_client_Id loaded: 02e0c255-a1a0-4d88-9868-ccdf253c948a
> ```
> 
> Log of device 2:
> 
> ```
> NOTICE: PLEX.clientinfo: Unique device Id plex_client_Id loaded: 02e0c255-a1a0-4d88-9868-ccdf253c948a
> ```
> 
> These strings need to be different from each other.
> 
> ## Workaround:
> * Make sure both Kodi instances are not running anymore.
> * On one of the 2 devices, navigate to the [Kodi userdata directory](https://kodi.wiki/view/Userdata)
> * Edit the file `settings.xml` in the folder `addon_data/plugin.video.plexkodiconnect`
> * Change the string in this line:
> 
> ```
> <setting id="plex_client_Id">02e0c255-a1a0-4d88-9868-ccdf253c948a</setting>
> ```
> 
> to something like this (so just a changed last letter `a` --> `b`
> 
> ```
> <setting id="plex_client_Id">02e0c255-a1a0-4d88-9868-ccdf253c948b</setting>
> ```
> 
> * You may need to restart the Plex Media Server, don't know

can confirm this workaround works. however is there you can fix or automate the process of generating a new "plex_client_Id" ?
